cpp/src/decisiontree/quantile/quantile.cuh

```diff
@@ -39,6 +41,7 @@ __global__ void computeQuantilesSorted(T* quantiles,
   int tid          = threadIdx.x + blockIdx.x * blockDim.x;
   double bin_width = static_cast<double>(length) / n_bins;
   int index        = int(round((tid + 1) * bin_width)) - 1;
+  index            = min(max(0, index), length - 1);
   // Old way of computing quantiles. Kept here for comparison.
```

Comment:
Since there is no longer old way of computing quantiles, we can remove these comments line now.